python/ruff/__main__.py

```diff
@@ -3,6 +3,20 @@
 import sysconfig
 
 
+def get_path_parts(path: str) -> list[str]:
```

Comment:
we only do checks against the last 3 elements, so perhaps we should stop after 3?

Reply:
I'm ok with stopping at 3, not a fan of returning a tuple.